features/steps/core.py

```diff
@@ -182,6 +189,11 @@ def check_json_output_path(context, path, value):
     assert struct == value, struct
 
 
+@then('the output should be empty')
+def check_empty_output(context, text=None):
+    assert (text or context.text) is None
```

Comment:
You need to check `context.stdout_capture` here, not `context.text` or `text`. You don't need `text` at all actually.